src/features/projectsV2/ProjectSnippet/microComponents/ImageSection.tsx

```diff
@@ -45,9 +47,25 @@ const ImageSection = (props: ImageSectionProps) => {
       }`
     );
   };
+  const handleBackButton = useCallback((e: MouseEvent) => {
+    e.stopPropagation();
+    if (window.history.length > 0) {
+      window.history.back();
+    } else {
+      window.location.href = '/';
+    }
+  }, []);
```

Comment:
The back button logic does not seem right, perhaps this can be taken up in a separate PR though.

Reply:
will be handle in a separate PR

Reply:
Noted. Have made a note in https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2149 as well.